pandas-stubs/core/indexes/range.pyi

```diff
@@ -24,7 +24,7 @@ class RangeIndex(Index[int]):
     ): ...
     def __init__(
```

Comment:
Also quick note here, the pandas implementation does not have a `__init__` method so we may want to remove that from pandas-stubs, @Dr-Irv let me know if you think it is good to drop.

Reply:
https://github.com/pandas-dev/pandas/blob/2419343bfea5dba678146139ca9663d831c47b22/pandas/core/indexes/range.py#L150-L158

Reply:
You can drop __init__.